main.go

```diff
@@ -182,6 +187,14 @@ func main() {
 	case *showLatestFlag:
 		showLatestVersion(custBinPath, mirrorURL)
 
+	/* latest required by module version */
```

Comment:
Maybe a bit more obviousness might make sense (despite I;m not sure this option is what we need in general):
```suggestion
	/* install latest required by module version */
```

Reply:
It is added in the same style as other comments above, e.g.
```
	/* latest stable version */
```

Reply:
Got it. Thanks.